### ISSUE TITLE
Error compiling refinery-migrations

### ISSUE THREAD

**Issue by glueball** [rust-db/refinery#64]
I get the following compiler error while trying to compile a "hello world" project using mysql_async:

```
Compiling refinery-migrations v0.2.0
error: expected an item keyword
   --> C:\Users\####\.cargo\registry\src\github.com-1ecc6299db9ec823\refinery-migrations-0.2.0\src\config.rs:259:21
    |
259 |                     let url = build_db_url("mysql", &config);
    |                     ^^^

error: aborting due to previous error

error: could not compile `refinery-migrations`.

To learn more, run the command again with --verbose.
```

Please, find attached a minimal example that shows the error. Doing a cargo build (with up to date stable rust) produces the given error.

[test.zip](https://github.com/rust-db/refinery/files/4249222/test.zip)

**Comment by jxs** [comment]
Hi, yeah it's a problem with [`cfg-if`](https://github.com/alexcrichton/cfg-if/issues/33) that seems to be introduced by `lexical`. I have released a new version `0.2.1` which locks `cfg-if` to 0.10.1, your example should work now
thanks for reporting!

**Comment by glueball** [comment]
Perfect. Working now!
